components/tabs/tab-panel-mixin.js

```diff
@@ -40,6 +45,11 @@ export const TabPanelMixin = superclass => class extends superclass {
 			:host([selected]) {
 				display: block;
 			}
+			:host(.focus-visible) {
+				outline: none;
+				border-radius: 0.3rem;
+				box-shadow: 0 0 0 2px var(--d2l-color-celestine);
+			}
```

Comment:
Tab panels [should](https://www.w3.org/WAI/ARIA/apg/patterns/tabs/) be focusable if the first element within the panel is not focusable.
>When the tab list contains the focus, moves focus to the next element in the page tab sequence outside the tablist, which is the tabpanel unless the first element containing meaningful content inside the tabpanel is focusable.